### DISCUSSION TITLE
restore whatsapp databases with `messages` table or `message` table

### DISCUSSION THREAD

**Discussion by menotuu** [KnugiHK/WhatsApp-Chat-Exporter#34] [Q&A]
I have problem to restore any of my database backups on android.
my database contains the `message` table form (seems to be the older format)and I cant restore it in official whatsapp way.
today I have to wait 48 hours to verificate my phonenumber just for `1` try to restore.

with Whatsapp-Chat-Exporter i can read the messages without a problem.

maybe I cant restore it beause its the older `message` table database style?
can anybody confirm same problem?

**Comment by KnugiHK** [comment]
As per discussion in #9, `message` table should be the new schema.
